macros/code.asm

```diff
@@ -1,11 +1,11 @@
 ; Syntactic sugar macros
 
 lb: MACRO ; r, hi, lo
-	ld \1, (\2 & $ff) << 8 + (\3 & $ff)
+	ld \1, (((\2) & $ff) << 8) | (((\3) & $ff))
```

Comment:
You don't need the parentheses enclosing the whole thing, right? `ld \1, (X)` → `ld \1, X`; same for in `ln`.